project/client/js/google-auth.js

```diff
@@ -0,0 +1,42 @@
+var googleUser = {};
+var initGoogleAuth = function() {
+    gapi.load('auth2', function(){
+        // Retrieve the singleton for the GoogleAuth library and set up the client.
+        auth2 = gapi.auth2.init({
+            client_id: '467471910617-vo4g6ojobn4o2tu25vo38ofurfis0jqj.apps.googleusercontent.com',
```

Comment:
What's this id? Does it need generating for play.benbru.com and should it be served to the frontend via some config?